chcemvediet/apps/inforequests/templates/inforequests/detail/action/main.html

```diff
@@ -14,8 +14,9 @@
 {% endcomment %}
 
 
-<div id="a{{ action.pk }}">
-  {% with print_id=idgenerator.next %}
+{% with print_id=idgenerator.next|format:"print-%s" %}
```

Comment:
@karol-bujacek Ten `|format:"print-%s"` treba vobec? Nestaci proste `{% with print_id=idgenerator.next %}`? Teda ze HTML ID elementu bude iba cislo vygenerovane z `idgenerator.next`?

Reply:
Netreba, tak isto, ako by nebolo treba `a` pri `id=a{{ action.pk }}`. Považoval som za lepšie to tam nedať, ale keď to robí takéto komplikácie, tak to dám radšej preč.